modules/aruco/test/test_aruco_utils.hpp

```diff
@@ -0,0 +1,128 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+namespace opencv_test {
+namespace {
+
+static inline vector<Point2f> getAxis(InputArray _cameraMatrix, InputArray _distCoeffs,
+                                      InputArray _rvec, InputArray _tvec, float length,
+                                            const float offset = 0.f)
+{
+    vector<Point3f> axis;
+    axis.push_back(Point3f(offset, offset, 0.f));
+    axis.push_back(Point3f(length+offset, offset, 0.f));
+    axis.push_back(Point3f(offset, length+offset, 0.f));
+    axis.push_back(Point3f(offset, offset, -length));
```

Comment:
The same, the result axis direction is opposite to what standard drawFrameAxes does.

Reply:
fixed